get.sh

```diff
@@ -87,13 +87,17 @@ getBinaryOpenjdk()
 	fi
 	jarDir=`ls -d */`
 	dirName=${jarDir%?}
-	mv $dirName j2sdk-image
+	if [ "$dirName" != "j2sdk-image" ]; then
+		mv $dirName j2sdk-image
+	else
+		echo "dirName is equal to j2sdk-image, skip moving"
+	fi
 }
 
 getTestKitGen()
 {
```

Comment:
If some other test project has retrieved testkitgen already, then we could skip this step.  It would be good to check first if the clone is required.

Reply:
@ShelleyLambert For jenkins story sdk is needed for all projects. When build is triggered by upstream sdk download is skipped.

For local running parameter SDKDIR is used to check if sdk is needed to download.
```
if [[ "$SDKDIR" != "" ]]; then
	getBinaryOpenjdk
fi

```

Reply:
@TianyuZuo  this should close issue #262

Reply:
Updated to check if TestConfig dir exists.